airline_routes/v1/get/cURL/airline_routes.sh

```diff
@@ -0,0 +1,8 @@
+# Authentication: $AMADEUS_CLIENT_ID & $AMADEUS_CLIENT_SECRET can be defined
+# in your environmental variables or directly in your script
+ACCESS_TOKEN=$(curl -H "Content-Type: application/x-www-form-urlencoded" \
+https://test.api.amadeus.com/v1/security/oauth2/token \
+-d "grant_type=client_credentials&client_id=$AMADEUS_CLIENT_ID&client_secret=$AMADEUS_CLIENT_SECRET" \
+| grep access_token | sed 's/"access_token": "\(.*\)"\,/\1/' | tr -d '[:space:]')
+
+curl -X GET "https://test.api.amadeus.com/v1/airline/destinations?airlineCode='BA'" -H "Authorization: Bearer $ACCESS_TOKEN" -k
```

Comment:
Should the parameter ```BA``` need to be surrounded by quotes in the url?

Reply:
@xianqiliu wow thanks for the catch! I just updated it :)